cincinnati/src/plugins/internal/cincinnati_graph_fetch.rs

```diff
@@ -81,10 +84,15 @@ impl CincinnatiGraphFetchPlugin {
             registry.register(Box::new(http_upstream_errors_total.clone()))?;
         };
 
+        let client = reqwest::ClientBuilder::new()
+            .build()
+            .map_err(|e| GraphError::FailedUpstreamRequest(e.to_string()))?;
```

Comment:
This is probably a wrong error to throw - any suggestions?

Reply:
This happens at initialization time before any plugin is executed. Other plugins don't set s a specific error here. How about a simple
```suggestion
            .context("Building reqwest client")?;
```

Reply:
Somehow my suggested change seems to be reverted again.

```suggestion
            .context("Building reqwest client")?;
```

Reply:
Sorry, must have overwritten that with a force push. Implemented that, thanks!